plugins/modules/ntnx_restore_protected_resources_v2.py

```diff
@@ -0,0 +1,236 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2025, Nutanix
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+DOCUMENTATION = r"""
+---
+module: ntnx_restore_protected_resources_v2
+short_description: Module to restore a protected resource in Nutanix Prism Central.
+description:
+  - This module can be used to restore a protected resource in Nutanix Prism Central.
+  - Supported only for protected resources that have minutely scheduled protection policies.
+options:
+  ext_id:
+    description:
+      - The external identifier of a protected VM or volume group.
+    type: str
+    required: true
+  cluster_ext_id:
+    description:
+      - The external identifier of the cluster on which the entity has valid restorable time ranges.
+      - The restored entity is created on the same cluster.
+    type: str
+    required: false
+  restore_time:
+    description:
+      - UTC date and time in ISO 8601 format representing the time from when the state of the entity should be restored.
+      - This must be a valid time within the restorable time range(s) for the protected resource.
+    type: str
+    required: false
+  wait:
+    description:
+      - Wait for the task to complete.
+    type: bool
+    required: false
+    default: True
+extends_documentation_fragment:
+  - nutanix.ncp.ntnx_credentials
+  - nutanix.ncp.ntnx_operations_v2
+author:
+  - George Ghawali (@george-ghawali)
+"""
+
+EXAMPLES = r"""
+- name: Restore VM
+  nutanix.ncp.ntnx_restore_protected_resources_v2:
+    nutanix_host: "{{ ip }}"
+    nutanix_username: "{{ username }}"
+    nutanix_password: "{{ password }}"
+    validate_certs: false
+    ext_id: "1ca2963d-77b6-453a-ae23-2c19e7a954a3"
+    cluster_ext_id: "00062aa9-1234-1122-3333-ac1f6b6f97e2"
```

Comment:
Give restorable time example as well. What format it should have also should be stated in doc